src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimContentProvider.java

```diff
@@ -5,6 +5,7 @@
 import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimantContentProvider;
 import uk.gov.hmcts.cmc.claimstore.services.staff.content.PersonContentProvider;
 import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
```

Comment:
resolve conflicts and please attach sample PDF

Reply:
[000MC031-sealed-claim.pdf](https://github.com/hmcts/cmc-claim-store/files/1820037/000MC031-sealed-claim.pdf)
[000MC029-claim-receipt (1).pdf](https://github.com/hmcts/cmc-claim-store/files/1820038/000MC029-claim-receipt.1.pdf)